diffkemp/simpll/ModuleComparator.cpp

```diff
@@ -140,8 +162,18 @@ void ModuleComparator::compareFunctions(Function *FirstFun,
                                                        showAsmDiffs,
                                                        &GS, DI, this);
             if (fCompSecond.compare() == 0) {
+                DEBUG_WITH_TYPE(DEBUG_SIMPLL,
+                    decreaseDebugIndentLevel();
+                    dbgs() << getDebugIndent()
+                           << "Inlined functions are equal\n"
```

Comment:
Better message: "After inlining, the functions are equal".
We're still comparing the same functions, just with some function calls within them inlined. We're not comparing the functions that were inlined.